prime-router/settings/prod/0163-nv-prod-up-onboarding.yml

```diff
@@ -0,0 +1,109 @@
+- name: "nv-phd"
+  description: "Nevada Public Health Department"
+  jurisdiction: "STATE"
+  stateCode: "NV"
+  countyName: null
+  filters: []
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:    
+  - name: "full-elr"
+    organizationName: "nv-phd"
+    topic: "full-elr"
+    customerStatus: "inactive"
+    translation: !<HL7>
+      schemaName: "metadata/hl7_mapping/receivers/STLTs/NV/NV-receiver-transform"
+      useTestProcessingMode: false
+      useBatchHeaders: true
+      receivingApplicationName: "NVDOH"
```

Comment:
Doesn't this go into the transform?

Reply:
@adegolier, thank you and yes, I have it in the transform as well.  I am updating as following:
![image](https://github.com/CDCgov/prime-reportstream/assets/20068283/ab1ae5f1-0582-4eaa-92d3-cf2eb2d63aff)